src/multicellular_stage/MulticellularSpecies.cs

```diff
@@ -210,6 +269,20 @@ public override object Clone()
         return result;
     }
 
+    public override ulong GetVisualHashCode()
+    {
+        ulong hash = 1099511628211;
+
+        foreach (var cell in Cells)
+        {
+            hash ^= cell.GetVisualHashCode() ^ (ulong)cell.Position.GetHashCode();
+
+            hash = (hash << 7) | (hash >> 57);
```

Comment:
Doesn't this mean that the order of the cells in the list affects the hash? That seems incorrect to me as the order of the cells shouldn't affect the visuals as they are all eventually placed anyway...